tailwind.config.js

```diff
@@ -1,12 +1,8 @@
 module.exports = {
-  theme: {
-  },
-  variants: {
-    boxShadow: ['group-hover'],
-    width: ['responsive', 'focus'],
-  },
+  theme: [],
+  variants: { boxShadow: ['group-hover'], width: ['responsive', 'focus'] },
```

Comment:
It looks like `variants` also has an `extend` in 2.x. I wonder if we could review/optimize these...

Reply:
Great idea!